packages/engine/lib/stateful/src/agent/schema.rs

```diff
@@ -5,7 +5,7 @@ use memory::arrow::{meta, meta::conversion::HashStaticMeta};
 
 use crate::{error::Result, field::FieldSpecMap};
 
-/// Describes the memory format used for storing [`Agent`]s.
+/// Describes the memory format for every [`Agent`]-containing [`AgentBatch`].
```

Comment:
I think this is even more confusing 😂 

Should we just say every [`AgentBatch`]